shippable.jobs.yml

```diff
@@ -16,6 +16,8 @@ jobs:
       - IN: dmsk_service_def
       - IN: dmsk_gitRepo
       - IN: dmsk_cliConfig
+        scopes:
+          - gke
```

Comment:
required to configure the gcloud sdk to use the region specified in your `dmsk_cliConfig` pointer region section